e2eRunner/src/test/scala/za/co/absa/hermes/e2eRunner/E2ERunnerJobSuite.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.hermes.e2eRunner
+
+import org.scalatest.FunSuite
+import za.co.absa.atum.utils.SparkTestBase
+
+class E2ERunnerJobSuite extends FunSuite with SparkTestBase {
+//  test("alfa") {
+//    E2ERunnerJob.main(Array(getClass.getResource("/TestDefinitionBase.json").toString))
+//  }
```

Comment:
Is this intended?

Reply:
A placeholder. I still have some UnitTests missing. Will fix this and add more